app/user.go

```diff
@@ -1691,9 +1692,44 @@ func (a *App) GetUserAnalytics(userId string, teamId string, name string) (*mode
 		// Return and exit from GetUserAnalytics with error
 		return nil, userLearningGroups.Err
 	} else {
+
+		userLearningGroupsQueryRows := userLearningGroups.Data.([]*model.LearningGroupQueryRow)
+		var userLearningGroups []*model.LearningGroup = make([]*model.LearningGroup, len(userLearningGroupsQueryRows))
```

Comment:
Don't repurpose a variable. In this case I think the previous use of this variable `userLearningGroups` should be given a different name, perhaps `userLearningGroupsQueryResult`?